docs/make.jl

```diff
@@ -179,5 +179,6 @@ makedocs(
          ])
 
 deploydocs(repo = "github.com/trixi-framework/Trixi.jl",
+           deploy_repo = "github.com/trixi-framework/TrixiDocumentation",
            devbranch = "main",
            push_preview = true)
```

Comment:
Can you also set `repo_previews` and use the ENV hack described in the current version to make it work with the current version of Documenter.jl?
Depending on how long it takes to publish the new version of Documenter.jl, this may be a workaround for us.

Reply:
I am unsure if that would fix it. By default, `repo_previews` is initialized to `repo`. 

The issue with the ENV hack is that when we decide to deploy a preview we look at the environment variable to determine if we are a PR to that repo.

I could probably hack something together where we use a different invocation for previews than normal builds.

Reply:
1f8dbfffe8b8d6fd544709b9edc4e75bd317070c is my attempt at that

Reply:
Let's see whether it works.